website/siteConfig.js

```diff
@@ -66,6 +66,12 @@ const siteConfig = {
     theme: 'default',
   },
 
+  markdownPlugins: [
+    function foo(md) {
+      md.inline.ruler.enable(['sub', 'sup']);
```

Comment:
Is this supposed to be in here? What does it do?

Reply:
Makes superscript & subscript work using ^ & ~ respectively
https://docusaurus.io/docs/en/site-config#markdownplugins-array

Reply:
Could the function be named a bit more helpfully?

Reply:
Open to suggestions...

Reply:
enableSubscriptAndSuperscript?

Reply:
Resolved in d4def37